playbooks/tasks/portal-role-task.yml

```
@@ -361,6 +361,15 @@
 
   when: webportal_setup_health_checks
 
+# Setup logrotate for webportal logs
+- block:
+    - name: Remove all skynet-webportal logrotate configs
+      command: rm -f /etc/logrotate.d/skynet-webportal-*
+
```

We need root/sudo for his:

```suggestion
      become: True
      
```